prov/sockets/src/sock_util.h

```diff
@@ -46,7 +46,7 @@ extern int sock_pe_waittime;
 extern long sock_dgram_drop_rate;
 #endif
 
-#define _SOCK_LOG_INFO(subsys, ...) FI_INFO(&sock_prov, subsys, __VA_ARGS__);
+#define _SOCK_LOG_INFO(subsys, ...) FI_DBG(&sock_prov, subsys, __VA_ARGS__);
```

Comment:
Mapping sock info to fi idbg rather than fi info is confusing.  Please rename SOCK_LOG_INFO to SOCK_LOG_DBG